config/initializers/0_clowder-config.rb

```diff
@@ -1,19 +1,33 @@
 
+def build_endpoint_url ep
+  config = ClowderCommonRuby::Config.load
+  uri = if config.tls
```

Comment:
You can just simply return from the condition, then you won't need the `else` and the variable allocation.